src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -93,6 +96,48 @@ describe('Test suite for dictionary concepts components', () => {
     instance.updateEventListener(event);
   });
 
+  it('should call updateEventListener function with internal mapping', () => {
+    const event = {
+      target: {
+        tabIndex: 0,
+        name: 'to_concept_name',
+        value: INTERNAL_MAPPING_DEFAULT_SOURCE,
+      },
+    };
+    const instance = wrapper.find('CreateConcept').instance();
```

Comment:
Shouldn't we expect something here?

Reply:
I have corrected this as well. Sorry about that.